test/over_react/component_declaration/component_base_test.dart

```diff
@@ -806,6 +806,101 @@ main() {
       }, timeout: new Timeout(const Duration(milliseconds: 250)));
     });
 
+
+    group('UiComponent2', () {
```

Comment:
We should also copy over the tests for the deprecated `copyUnconsumedProps` / `copyUnconsumedDomProps` functions so that they are being tested with a `UiComponent2` instance.

Reply:
Yep, good thought!